actors.md

```diff
@@ -708,3 +708,233 @@ func UpdatePeerID(pid PeerID) {
 ## Payment Channel Broker Actor
 
 TODO
+
+## Multisig Account Actor
+
+A basic multisig account actor. Allows sending of messages like a normal account actor, but with the requirement of M of N parties agreeing to the operation. Any party can veto any operation (which is a potentially debateable design decision). Completed and/or cancelled operations stick around in the actors state until explicitly cleared out.
+
+#### State
+
+```go
+type Multisig struct {
+    Signers []Address
+    Required uint
+
+    NextTxID uint64
+    Transactions map[int]Transaction
+}
+
+type Transaction struct {
+    TxID uint64
+    To Address
+    Value TokenAmount
+    Method string
+    Params []byte
+    Approved []Address
+    Completed bool
+    Canceled bool
+}
+```
```

Comment:
It would be nice to have a method that queries the current state of a transaction, so I can know if I need/want to do sth about it

Reply:
On that note, rather than have 
`Completed bool
Canceled bool`

We might just have a State int, with a few potential states: ongoing, completed, canceled...